src/commands/edge_app.rs

```diff
@@ -483,26 +470,26 @@ impl EdgeAppCommand {
         Ok(())
     }
 
-    fn get_root_edge_app_asset(&self, manifest: &EdgeAppManifest) -> Result<String, CommandError> {
+    fn check_installation_exists(&self, manifest: &EdgeAppManifest) -> Result<(), CommandError> {
```

Comment:
well it should be get_installation then because we are not going to store the installation id in the file

Reply:
It doesn't actually return installation, as we discussed that we don't want to store it.
So what this function actually does is it checks if installation exists or return error. check or may be ensure fits it.

Reply:
In this PR: https://github.com/Screenly/cli/pull/67
renamed it to get_installation as I need it there.